ext/spl/php_spl.stub.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+function iterator_to_array(Traversable $iterator, bool $use_keys = true): array {}
+
+function iterator_count(Traversable $iterator): int {}
+
+function iterator_apply(Traversable $iterator, callable $function, ?array $args = []): int {}
+
+/**
+ * @param object|string $instance
+ * @return array|false
+ */
+function class_parents($instance, bool $autoload = true) {}
+
+/**
+ * @param object|string $what
+ * @return array|false
+ */
+function class_implements($what, bool $autoload = true) {}
+
+/**
+ * @param object|string $what
+ * @return array|false
+ */
+function class_uses($what, bool $autoload = true) {}
+
+function spl_classes(): array {}
+
+/** @return array|false */
+function spl_autoload_functions() {}
+
+function spl_autoload(string $class_name, string $file_extensions): void {}
+
+function spl_autoload_extensions(string $file_extensions): string {}
```

Comment:
`$file_extensions` is optional:
```suggestion
function spl_autoload_extensions(string $file_extensions = UNKNOWN): string {}
```